src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -905,8 +905,12 @@ public void setMinImageSize(int min) {
           defaultValue = "XYZCT"
   )
   public void setDimensionOrder(DimensionOrder order) {
-    LOGGER.warn(
-      "--dimension-order is deprecated; OME-NGFF requires XYZCT output order");
+    if (order != DimensionOrder.XYZCT) {
+      LOGGER.warn(
+        "--dimension-order is deprecated;" +
+        " OME-NGFF requires XYZCT output order");
+      dimensionOrder = order;
```

Comment:
This line could be removed given that `dimensionOrder` will be set in line 914 anyway?